src/plugins/discover/public/application/main/hooks/utils/build_state_subscribe.test.ts

```diff
@@ -24,7 +24,7 @@ describe('buildStateSubscribe', () => {
       appState: stateContainer.appState,
       savedSearchState: stateContainer.savedSearchState,
       dataState: stateContainer.dataState,
-      internalState: stateContainer.internalState,
+      internalState: stateContainer.sharedState,
```

Comment:
```suggestion
      sharedState: stateContainer.sharedState,
```
Shouldn't it be updated too?

Reply:
definitely, well apply and align those suggestions 👍  thx